### ISSUE TITLE
403 Access denied

### ISSUE THREAD

**Issue by LalleSX** [romgrk/node-gtk#346]
Is it over?

```js
node-pre-gyp ERR! install response status 403 Forbidden on https://node-gtk-1.s3.amazonaws.com/node_gtk/v0.11.0/node_gtk-node-v111-linux-x64.tar.gz
```
Idk who else to report this to, so im reporting it here.

This happens when trying to install via ´npm´ 

Hope this helps :smile:

**Comment by systemd11** [comment]
Have  a similar problem.

**Comment by romgrk** [comment]
I have published v0.12.0 recently, with pre-built binaries for node 16 & 18 on linux & windows only. macOS builds have been unreliable.

https://github.com/romgrk/node-gtk/issues/347#issuecomment-1610871105

If you still have this issue re-open an issue and list your environment (OS & versions).
